jquery.flot.js

```diff
@@ -2120,7 +2120,7 @@ Licensed under the MIT license.
 
         function drawSeries(series) {
             if (series.lines.show)
-                $.plot.drawSeries.drawSeriesLines(series, ctx, plotOffset, plotHeight, getColorOrGradient);
+                $.plot.drawSeries.drawSeriesLines(series, ctx, plotOffset, plotWidth, plotHeight, getColorOrGradient);
```

Comment:
you better pass the same set or params to all of them. We will eventually need them in all the draw plugins